src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,42 @@
+import React from 'react';
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
+  <div className="card" data-cy="Movie">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={movie.imgUrl}
+          alt="Film logo"
```

Comment:
The alt attribute for the img tag should be more descriptive, ideally related to the movie itself for better accessibility. You can use the movie's title for the alt attribute of the img tag.